XamarinCommunityToolkitSample/Pages/Views/AvatarViewPage.xaml

```diff
@@ -22,7 +21,7 @@
                                      Orientation="Horizontal"
                                      Spacing="10">
 
-                            <views:AvatarView ColorTheme="{x:Static views:ColorTheme.Jungle}"
+                            <xct:AvatarView ColorTheme="{x:Static xct:ColorTheme.Jungle}"
```

Comment:
You need to change the `views` to `xct`
```xml
    <views:AvatarView.Source>
            <UriImageSource Uri="{Binding Source}" />
    </views:AvatarView.Source>
</views:AvatarView>

Reply:
good catch... not sure how my local build didn't fail